### ISSUE TITLE
Floating point exception(core dumped) 

### ISSUE THREAD

**Issue by FinSalMar** [weizhouUMICH/SAIGE#30]
I receive this error when trying to run Step 1 with LOCO=TRUE.

> Leave-one-chromosome-out is activated
> chromosomeStartIndexVec:  0 3922 5051 7056 9077 10853 12642 13835 14336 17521 20104 22092 24181 26491 28662 30754 32575 33590 35109 36783 38618 39973 
> 11115  samples have genotypes
> formula is  phen~ 1 
> 11115  samples have non-missing phenotypes
> 11115  samples will be used for analysis
> colnames(data.new) is  Y minus1 
> out.transform$Param.transform$qrr:  1 1 
> phen  is a binary trait
> 
> Call:  glm(formula = formula.new, family = binomial, data = data.new)
> 
> Coefficients:
> minus1  
>  1.461  
> 
> Degrees of Freedom: 11115 Total (i.e. Null);  11114 Residual
> Null Deviance:	    15410 
> Residual Deviance: 10750 	AIC: 10760
> [1] "Start reading genotype plink file here"
> nbyte: 2779
> nbyte: 2779
> reserve: 115550544
> /var/tmp/slurmd/job10762729/slurm_script: line 23: 13914 Floating point exception(core dumped) Rscript Step1_new.R --plinkFile=plinkfile --phenoFile=file.phen --phenoCol=phen --sampleIDColinphenoFile=id --numMarkers=100 --traitType=binary --outputPrefix=output --nThreads=1
> 

**Input is as follows:**

```
Rscript Step1_new.R \
	--plinkFile=plinkfile \
	--phenoFile=file.phen \
	--phenoCol=phen \
	--sampleIDColinphenoFile=id \
	--numMarkers=100 \
	--traitType=binary \
	--outputPrefix=output \
	--nThreads=1
```

**Comment by weizhouUMICH** [comment]
Did you see the error when LOCO=FALSE?

**Comment by mswaters** [comment]
Yes, I receive the same error.

**Comment by weizhouUMICH** [comment]
Hi @FinSalMar, 
Could you please try the newest version to see if the error persists? I'm so sorry for taking so long to get back to you. Please reopen this issue if the error is still seen. 

Thanks,
Wei

**Comment by laurent-ntnu** [comment]
Hi Wei,

I also got the same error in step 1: Floating point exception (core dumped).

The problem comes from a division by 0 in file: `src/SAIGE_fitGLMM_fast.cpp`
when all individuals in the subset have missing data for a given variant.

The line with the error is:
`freq = sum(m_OneSNP_Geno)/float((2*(Nnomissing-lengthIndexNA)));`

It fails when `Nnomissing == lengthIndexNA`.

This happend to me when running a GWAS on females, where the genotype file included Y chromosome. 
A quick work around was to remove chrY from the genotype file.

But it would be nice if the program could automatically skip variants where:
`Nnomissing == lengthIndexNA`

Thanks

**Comment by weizhouUMICH** [comment]
Hi Laurent, 

Thanks for reporting this! This issue should has been fixed since the version 0.36.4. 
https://github.com/weizhouUMICH/SAIGE/issues/154

Could you please try the newest version 0.36.5? 
https://github.com/weizhouUMICH/SAIGE#log-for-fixing-bugs

Thanks
Wei

**Comment by laurent-ntnu** [comment]
Yes it is working with version 0.36.5.
Thanks
